### ISSUE TITLE
[Needs assistive message feature] Order Confirmation needs vocalized feedback

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13587]
**Describe the bug**

- There is no verbal confirmation alert that a transaction has happened. Before actually landing on the confirmation page.
- Once on the Confirmation page, there's no feedback, (this is mainly because the Page title is not being read).

**Tell us the version of Spartacus**
- Library version: 4.0.0.

**To Reproduce**
1. Open any Screen Reader (VO, NVDA, JAWS).
2. Add products to cart.
3. Checkout products.
4. Hear absence of feedback.

**Expected behavior**
- Right after hitting the **Place Order** button, Screen Reader vocalizes: _"Thank you for your order! An invoice has been sent by email. Redirecting to the Confirmation page."_
- When landing on the Confirmation page, Screen Reader vocalizes: _"Order Confirmation, Confirmation of Order 00000012"_
- Use: `aria-live="polite/assertive"` and `aria-atomic="true"`.

**Screenshots**
![Screen Shot 2021-08-31 at 3 04 54 PM](https://user-images.githubusercontent.com/15113219/131561582-aeeb5907-1440-4cb5-9950-da2f20799a8c.png)

**Comment by BenjaminEmiliani** [comment]
On hold until #14371

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.

When we click on "Place Order", we get a message saying "Order is in Process"
![OrderPlaced](https://user-images.githubusercontent.com/45409515/157745766-fca9207d-0d9e-4c8a-a51b-5c7811e7a8dc.png)

Once it lands on the Order Confirmation Page, we get the message about the order confirmation and invoice.
![OrderConf2](https://user-images.githubusercontent.com/45409515/157745778-048631e6-92d5-4484-8187-3fb485b7e087.png)
![OrderConf1](https://user-images.githubusercontent.com/45409515/157745779-78e2f97f-33bc-4152-ba1f-79291ceaf89f.png)
